helm/hpcc/templates/eclagent.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         accessDali: "yes"
         accessEsp: {{ .useChildProcesses | default false | ternary "yes" "no" | quote }}
     spec:
-      {{- include "hpcc.placementMatchName" (dict "root" $ "pod" .name "cluster" .name "type" "eclagent" ) | indent 6 }}
+            {{- include "hpcc.placementsByPodTargetType" (dict "root" $ "pod" .name "target" .name "type" "eclagent" ) | indent 6 }}
```

Comment:
appears to be an indentation/formatting issue, tabs?

Reply:
fixed